src/com/hms_networks/americas/sc/localdatafiles/LocalDataFileManager.java

```diff
@@ -310,6 +310,10 @@ public synchronized void addDataPoint(DataPoint dataPoint, boolean overwrite) {
    * Appends the specified alarm point to the buffer which is written to local data file storage
    * every {@link #fileWriteIntervalMillis} milliseconds.
    *
+   * @throws IllegalStateException when tag list has not been populated with {@link
+   *     TagInfoManager#refreshTagList()}
+   * @throws IndexOutOfBoundsException when (tagId - {@link TagInfoManager#lowestTagIdSeen()}) is not
```

Comment:
```suggestion
   * @throws IndexOutOfBoundsException when (tagId - {@link TagInfoManager#getLowestTagIdSeen()}) is not
```

Not fixed. lowestTagIdSeen() is also not valid. #lowestTagIdSeen is a variable, not function/method.